docs/examples/prototypes.md

```diff
@@ -65,26 +65,152 @@ amadeus-travel-restrictions-node ([Link to GitHub](https://github.com/amadeus4de
+
+
+### amadeus-trip-purpose-django
+
+This project ([Link to GitHub](https://github.com/amadeus4dev-examples/amadeus-trip-purpose-django){:target="\_blank"}) demonstrates how to integrate Amadeus APIs using the [Python SDK](../programming/python.md) in a Django application. The end user submits round-trip information via a form and the [Trip Purpose Prediction](https://developers.amadeus.com/self-service/category/trip/api-doc/trip-purpose-prediction){:target="\_blank"} is called. This API predicts if the given journey is for leisure or business purposes.
+
+![amadeus-trip-purpose-django](../images/amadeus-trip-purpose-django.png)
+
+### amadeus-hotel-search-swift
+This prototype ([Link to GitHub](https://github.com/amadeus4dev-examples/amadeus-hotel-search-swift){:target="\_blank"}) demonstrates a simple iOS hotel search app from scratch using Amadeus Hotel Search API (version 2.1 - decommissioned) and [iOS SDK](https://github.com/amadeus4dev-examples/amadeus-ios){:target="\_blank"}.
+
+![amadeus-hotel-search-swift](../images/amadeus-hotel-search-swift.png)
+
+### amadeus-hotel-booking-android
+This prototype ([Link to GitHub](https://github.com/amadeus4dev-examples/amadeus-hotel-booking-android){:target="\_blank"}) shows how to use the [Android SDK](https://github.com/amadeus4dev-examples/amadeus-android){:target="\_blank"} to build a Hotel Booking Engine in Android Studio. 
+
+![amadeus-hotel-booking-android](../images/amadeus-hotel-booking-android.png)
+
+
+###  amadeus-smart-flight-search-django
+This prototype ([Link to GitHub](https://github.com/amadeus4dev-examples/amadeus-smart-flight-search-django){:target="\_blank"}) shows how the Air APIs can be integrated with the Django framework and [Python SDK](../programming/python.md), by calling the [Flight Choice Prediction](https://developers.amadeus.com/self-service/category/air/api-doc/flight-choice-prediction){:target="\_blank"} and [Trip Purpose Prediction](https://developers.amadeus.com/self-service/category/trip/api-doc/trip-purpose-prediction){:target="\_blank"}. We also call the [Flight Offers Search](https://developers.amadeus.com/self-service/category/air/api-doc/flight-offers-search){:target="\_blank"} as a more traditional method of flight search and we compare its results with the [Flight Choice Prediction](https://developers.amadeus.com/self-service/category/air/api-doc/flight-choice-prediction){:target="\_blank"} ones to show the power of AI.
+
+![amadeus-smart-flight-search-django](../images/amadeus-smart-flight-search-django.png)
+
+
+### amadeus-flight-search-wordpress-plugin
+
+This prototype ([Link to GitHub](https://github.com/amadeus4dev-examples/amadeus-flight-search-wordpress-plugin){:target="\_blank"})  demonstrated how to build a basic flight search feature using the [Flight Offers Search API](https://developers.amadeus.com/self-service/category/air/api-doc/flight-offers-search){:target="\_blank"} with WordPress plugin.
```

Comment:
I would say how to create a WordPress plugin to build a basic flight search. "with Wordpress plugin" is not so accurate because it sounds like we use it to make the search, but the purpose is to show developers how to build a WP plugin for flight search